docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -0,0 +1,239 @@
+---
+assignees:
+- janetkuo
+title: Performing a Rolling Update on a DaemonSet
+---
+
+{% capture overview %}
+
+This page shows how to perform a rolling update on a DaemonSet. 
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+* The DaemonSet rolling update feature is only supported in Kubernetes version 1.6 or later.
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## DaemonSet Update Strategy
+
+DaemonSet has two update strategy types :
+
+* OnDelete: This is the default update strategy for backward-compatibility. With
+  `OnDelete` update strategy, after you update a DaemonSet template, new
+  DaemonSet pods will *only* be created when you manually delete old DaemonSet
+  pods. This is the same behavior of DaemonSet in Kubernetes version 1.5 or
+  before.
+* RollingUpdate: With `RollingUpdate` update strategy, after you update a
+  DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
+  will be created automatically, in a controlled fashion. 
+
+## Limitations 
+
+* DaemonSet rollout history is not supported yet. 
+* DaemonSet rollback is not directly supported in `kubectl` yet. You can rollback
+  by updating DaemonSet template to match the previous version.
+* Every time you modify the `RollingUpdate` template for a DaemonSet, Kubernetes
```

Comment:
Not saying that we shouldn't optimize this to adopt the older pods but we probably shouldn't list it as a limitation.